package.json

```diff
@@ -80,6 +80,7 @@
     "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#main-artifacts",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@ethersproject/contracts": "^5.8.0",
```

Comment:
Explicitly set a version, otherwise it will be derived from `@safe-global/safe-core-sdk-types` as 5.7.2